testdata/errorf.go

```diff
@@ -21,4 +22,19 @@ func f(x int) error {
 	return nil
 }
 
+// TestF is a dummy test
+func TestF(t *testing.T) error {
+	x := 1
+	if x > 10 {
+		return t.Error(fmt.Sprintf("something %d", x)) // MATCH /should replace.*t\.Error\(fmt\.Sprintf\(\.\.\.\)\).*t\.Errorf\(\.\.\.\)/
```

Comment:
by specific, I mean that the regexp is tighter than it needs to be. but now that I think about it more, this is fine as is.